app/models/audit.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class Audit < Audited::Audit
+  scope :viewing_birth_record, ->() { where(comment: AuditedOperationsService::VIEW_SHARED_BIRTH_RECORD) }
```

Comment:
This is the biggest backend change - I'm a little uncomfortable about reaching into the service to get this constant, but I think it's probably the best at the moment